src/sincere_singularities/modules/order.py

```diff
@@ -28,10 +30,17 @@ class CustomerInfo:
 class Order:
     """The Dataclass Containing Order Information."""
 
+    order_timestamp: datetime | None = None
```

Comment:
I don't think we should include it here because it gets overwritten in `__post_init__` anyways